py3status/modules/uptime.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display the system's uptime
+
+Configuration parameters:
+        - format : what will be displayed, formatted with predefined
+          placeholders; defaults to "{days} days, {hours} hours, {minutes} min"
+
+Available placeholders in format string:
+        - {days}: days part of the uptime
+        - {hours}: hours part of the uptime
+        - {minutes}: minutes part of the uptime
+        - {seconds}: seconds part of the uptime
+        - No {year} provided !
```

Comment:
@Horgix If you add a year placeholder, I'd suggest doing it with an option to enable and otherwise leave just leave it with days. Compatibility with the `uptime` format is desirable here.

@tobes I'd argue about your "hygiene" comment but if you're running a WM then probably, yeah.